module/actor/coterie-actor-sheet.js

```diff
@@ -0,0 +1,246 @@
+/* global ActorSheet, game, mergeObject, duplicate */
```

Comment:
the coterie sheet has the most basic things :
Features, Notes, Equipement & Biography